src/components/Carousel/Carousel.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import Slider from "react-slick";
+import "../../../node_modules/slick-carousel/slick/slick.css"; 
+import "../../../node_modules/slick-carousel/slick/slick-theme.css";
+import './Carousel.css'
+import Hero from '../Hero/Hero.js'
+
+const Carousel = ({movies}) => {
+  const featuredFilms = movies.slice(10, 15).map(movie => {
```

Comment:
This is a great way to grab some of our mock data. We can adjust it later once we get the network requests working. And if we end up displaying or sorting by genre, maybe we can handpick a movie from each category to feature.